Add the new getting started content (#10596)
PostHog/posthog.com: contents/docs/new-to-posthog/understand-posthog.mdx

---
title: Understanding PostHog
---

import { Admonition } from 'components/Docs/Admonition'

PostHog is a lot of power, so to get the most from it, we should review how data is structured, grouped, and related.

> ![PostHog data flow](https://res.cloudinary.com/dmukukwp6/image/upload/posthog_data_diagram_84bf3b121b.png)

Reviewer: I think the diagram would actually be better at the bottom of the page – i.e. explain the concepts first, then bring in the diagram showing how they fit together.

PostHog's data model starts with **[events](/docs/data/events)**, single actions that a user triggered at a specific point in time. These are sent either from one of our [SDKs](/docs/libraries) or directly via our [API](/docs/api).

> Events are flexible: they can be captured automatically, via [autocapture](/docs/product-analytics/autocapture), or you can emit your own [custom events](/docs/getting-started/send-events), attaching additional metadata via [properties](/docs/data/events#event-properties).

Reviewer 2: ```suggestion
Events are flexible: they can be captured automatically via [autocapture](/docs/product-analytics/autocapture), or you can capture your own [custom events](/docs/getting-started/send-events) and attach additional metadata via [properties](/docs/data/events#event-properties).
```

You might create an event to represent purchasing an upgrade, with custom properties like `price` or `renewal_period`.

### Actions

[Actions](/docs/data/actions) are a layer on top of events. Define actions to aggregate multiple events, extract a specific kind of event according to its properties, or give an event a more useful name.

### Sessions

> A [session](/docs/data/sessions) is a stream of events corresponding to a specific usage session of a product. Sessions have properties, and you can query them.

Reviewer 2: ```suggestion
A [session](/docs/data/sessions) is a collection of events corresponding to a specific usage session of a product. Sessions have properties, and you can query them.
```

### Person profiles

Meanwhile, users of your product are given a **[person profile](/docs/data/persons)**, which gather these events. Person profiles similarly contain properties. Some are set automatically:

> - browser details
> - geo IP data
> - referrers

Reviewer 2: ```suggestion
- Browser details
- Geo IP data
- Referrers
```
